src/coupling/hc_explicit_sequential.cc

```diff
@@ -44,7 +39,8 @@
 
 
 FLOW123D_FORCE_LINK_IN_PARENT(transportOperatorSplitting);
-FLOW123D_FORCE_LINK_IN_PARENT(soluteTransport);
+FLOW123D_FORCE_LINK_IN_PARENT(soluteTransportDG);
```

Comment:
These macros force  correct linking of object files. Therefore we should use rather filename-like identifiers.